Mooch-Lightning/Model/User.cs

```diff
@@ -13,5 +13,11 @@ public class User
     public string ImageUrl { get; set; }
     public List<MoochRequest> MoochRequests { get; set; }
     public List<UserMembership> UserMemberships { get; set; }
```

Comment:
I think these should be nullable... If they aren't they'll be required when adding a new user

```suggestion
    public List<MoochRequest>? MoochRequests { get; set; }
    public List<UserMembership>? UserMemberships { get; set; }
```